bundles/org.openhab.binding.broadlinkthermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,136 @@
+
+		<channels>
+			<channel id="power" typeId="power"/>
+			<channel id="mode" typeId="mode"/>
+			<channel id="sensor" typeId="sensor"/>
+			<channel id="roomtemperature" typeId="roomtemperature"/>
+			<channel id="roomtemperatureexternalsensor" typeId="roomtemperatureexternalsensor"/>
+			<channel id="active" typeId="active"/>
+			<channel id="setpoint" typeId="setpoint"/>
+			<channel id="temperatureoffset" typeId="temperatureoffset"/>
+			<channel id="remotelock" typeId="remotelock"/>
+			<channel id="time" typeId="time"/>
+		</channels>
+
+		<representation-property>host</representation-property>
+
+		<config-description-ref uri="thing-type:broadlinkthermostat:floureonandhysenthermostat"/>
+	</thing-type>
+
+	<channel-type id="power">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Switch display on/off and enable/disables heating</description>
+		<category>Switch</category>
+	</channel-type>
+	<channel-type id="mode">
+		<item-type>String</item-type>
+		<label>Mode</label>
+		<description>Current mode of the thermostat</description>
+		<state>
+			<options>
+				<option value="auto">auto</option>
+				<option value="manual">manual</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="sensor">
+		<item-type>String</item-type>
+		<label>Sensor</label>
+		<description>The sensor (internal/external) used for triggering the thermostat</description>
+		<category>Sensor</category>
+		<state>
+			<options>
+				<option value="internal">internal</option>
+				<option value="external">external</option>
+				<option value="internal_temp_external_limit">internal_temp_external_limit</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="active">
+		<item-type>Switch</item-type>
+		<label>Active</label>
+		<description>Shows if thermostat is currently actively heating</description>
+		<category>Switch</category>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="roomtemperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Room Temperature</label>
+		<description>Room temperature, measured directly at the device</description>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="roomtemperatureexternalsensor">
+		<item-type>Number:Temperature</item-type>
+		<label>Room Temperature external sensor</label>
+		<description>Room temperature, measured by the external sensor</description>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="setpoint">
+		<item-type>Number:Temperature</item-type>
+		<label>Setpoint</label>
+		<description>Temperature setpoint that open/close valve</description>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" step="0.5"/>
+	</channel-type>
+	<channel-type id="temperatureoffset">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature offset</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Done.